go.mod

```diff
@@ -25,7 +26,7 @@ require (
 	cloud.google.com/go v0.81.0 // indirect
 	github.com/Azure/go-autorest v14.2.0+incompatible // indirect
 	github.com/Azure/go-autorest/autorest v0.11.18 // indirect
-	github.com/Azure/go-autorest/autorest/adal v0.9.13 // indirect
+	github.com/Azure/go-autorest/autorest/adal v0.9.22 // indirect
```

Comment:
We normally don't update the vendor package only if there's a specific reason

Reply:
Removed the update for the vendor package